plugins/tiddlywiki/filesystem/filesystemadaptor.js

```diff
@@ -75,7 +75,16 @@ FileSystemAdaptor.prototype.getTiddlerFileInfo = function(tiddler,callback) {
 /*
 Save a tiddler and invoke the callback with (err,adaptorInfo,revision)
 */
-FileSystemAdaptor.prototype.saveTiddler = function(tiddler,callback,options) {
+FileSystemAdaptor.prototype.saveTiddler = function(tiddler,options,callback) {
+	// Starting with 5.1.24, all syncadptor method signatures follow the node.js
```

Comment:
I'll do a search and replace for 5.1.24 -> 5.2.0 before release, but it might be worth updating it here for clarity. Also, it is preferred to prefix version numbers with a "v".